scenedetect/cli/__init__.py

```diff
@@ -613,6 +613,63 @@ def detect_threshold_command(
     )
 
 
+@click.command('detect-hash')
+@click.option(
+    '--threshold', '-t', metavar='VAL',
+    type=click.FLOAT, default=100.0, show_default=True, help=
+    'Threshold value (float) that the hash_dist metric must exceed to trigger'
+    ' a new scene. Refers to frame metric hash_dist in the stats file.'
+)
+@click.option(
+    '--size', '-s', metavar='VAL',
+    type=click.IntRange(min=2), default=16, show_default=True, help=
+    'Size of the hash used in the perceptual hasing algorithm. Must be an '
+    'integer >=2.'
+)
+@click.option(
+    '--freq_factor', '-f', metavar='VAL',
+    type=click.IntRange(min=1), default=2, show_default=True, help=
```

Comment:
Should re-use the value from `CONFIG_MAP` for `min` here and elsewhere.

Reply:
I think I addressed this in efa4023 (I based this off what you did in `detect-threshold`, but would love you to confirm.